python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -807,3 +851,266 @@ def explain_graph(self, graph, feat, tmp=1.0, training=False, **kwargs):
             self.clear_masks()
 
         return (probs, hetero_edge_mask)
+
+    def explain_node(
+        self, nodes, graph, feat, tmp=1.0, training=False, **kwargs
+    ):
+        r"""Learn and return an edge mask that plays a crucial role to
+        explain the prediction made by the GNN for provided set of node IDs.
+        Also, return the prediction made with the batched graph and edge mask.
+
+        Parameters
+        ----------
+        nodes : dict[str, interable[int]]
+            A dict mapping node types (keys) to an iterable set of node ids (values).
+        graph : DGLGraph
+            A heterogeneous graph.
+        feat : dict[str, Tensor]
+            A dict mapping node types (keys) to feature tensors (values).
+            The input features are of shape :math:`(N_t, D_t)`. :math:`N_t` is the
+            number of nodes for node type :math:`t`, and :math:`D_t` is the feature
+            size for node type :math:`t`
+        tmp : float
+            The temperature parameter fed to the sampling procedure.
+        training : bool
+            Training the explanation network.
+        kwargs : dict
+            Additional arguments passed to the GNN model.
+
+        Returns
+        -------
+        dict[str, Tensor]
```

Comment:
max char 80 including the prefix space for the python doc below except for the example code.

Reply:
updated